### ISSUE TITLE
GO enrichment widget - adding details

### ISSUE THREAD

**Issue by kkarra** [intermine/intermine#205]
+1 for this request, here's a note from a user via curator mail:

Hi, I really like the GO enrichment tool on Yeastmine, but one thing I miss is the ability to see the fold enrichment of each category, in addition to the p-value. That is, how many times more we see genes in the given category compared to the background frequency in the genome. /Anders"

Rama Balakrishnan wrote:

```
For each enriched term, can you request that the background frequency (x input genes/total # of background
genes) and cluster frequency (x genes/y # of input genes) numbers are displayed for this widget?
```

**Comment by julie-sullivan** [comment]
Fixed by #1601 

Added to web services, not UI (Yet)

**Comment by julie-sullivan** [comment]
Fixed by #1601
